e2e/steps_file.js

```diff
@@ -232,6 +178,18 @@ module.exports = function () {
         .withText('Remove'));
     },
 
+    async submitNewCaseWithData(filename = 'mandatorySubmissionFields') {
+      const caseId = await this.logInAndCreateCase(config.swanseaLocalAuthorityUserOne);
+      await this.signIn(config.systemUpdateUser);
+      await this.navigateToCaseDetails(caseId);
+      await caseViewPage.goToNewActions(config.applicationActions.populateCase);
+      populateCaseEventPage.setCaseDataFilename(filename);
+      await this.completeEvent('Submit');
```

Comment:
Just FMI (not related to this PR), Do you know why below is 
```
async submit(button) {
    await I.retryUntilExists(() => I.click(button), '.alert-success');
  },
```

instead of something like this

```
submit(button) {
    I.waitForElement('.alert-success');
    I.click(button);
  },
```
which would mean, we don't have do async await on every function call. There is nothing wrong with how its done here but I am wondering if all these awaits everywhere is really necessary.

Reply:
Looks like what i said above is possible with WebDriver but we use Puppeteer. I am not yet sure, how we can achieve same with Puppeteer. Still checking, I will get back if i find anything.

Reply:
Yup, I also noticed that, we do have quite a lot unnecessary awaits in e2e tests. Planned to check that outside of the scope of ticket, maybe we should raise another one.